src/views/catalog/CreateFromInstanceTypes/CreateFromInstanceType.scss

```diff
@@ -36,4 +42,7 @@
     --pf-c-divider--BorderWidth--base: var(--pf-global--spacer--form-element);
     --pf-c-divider--after--BackgroundColor: var(--pf-global--palette--black-200);
   }
+  &__HelpTextIcon {
+    max-width: 425px !important;
```

Comment:
same as above

Reply:
Without it, we are unable to achieve the expected look, unfortunately.

Reply:
I checked that for Firefox browser, it doesn't work without `!important`, and I haven't found any other way to achieve the expected look, except using a component prop `maxWidth` instead (but I was told not to do so, and better to style it in css file). However, in Chrome, it works without `!important`. 

Another way is to omit this styling totally, but the popover text container will be shorter and looking less nice, not as recommended via design doc.